index.html

```diff
@@ -31,7 +32,7 @@
       property="twitter:description"
       content=" Web3Collectives is a free library of over 200 web3 resources on varying topics like Smart contract deployment, building decentralized applications, cryptography fundementals and lots more..."
```

Comment:
Please use this for the description:
`Web3Collectives is a free Web3 resource library that helps you level up your blockchain development and technology skills by providing a wide range of learning materials, including popular and recommended resources such as interactive tutorials, how-to guides, courses, books, documentation, and videos.`